configure

```diff
@@ -685,6 +685,25 @@ if [ $os = "Linux" -o $os = "Darwin" ]; then
 
   foo=foo_$$
 
+  cat > ${foo}.F << EOF
+      PROGRAM GFORTRAN_VERSION_CHECK
+
+        IF (__GNUC__ .GT. 9) CALL EXIT(1)
+
+      END PROGRAM
+EOF
+
+  grep '^SFC' configure.wrf | grep -i 'gfortran' > /dev/null
+  if [ $? ]
+  then
+    gfortran -o ${foo} ${foo}.F > /dev/null 2>&1 && ./${foo}
+    if [ $? -eq 1 ]
+    then
+      sed '/^FCBASEOPTS_NO_G/ s/$/ -fallow-argument-mismatch -fallow-invalid-boz/' configure.wrf > configure.wrf.edit
+      mv configure.wrf.edit configure.wrf
+    fi
+  fi
+
```

Comment:
@syalavarthi 
Srikanath,
I modified your logic a bit. Instead of the second `if [ $? ]`, I swapped in `if [ $? -eq 1 ]`. The idea is to only add the additional arguments when the compiler is gcc/10 or later.

Would you verify that this still works for you?